art/api/models.py

```diff
@@ -1,3 +1,8 @@
 from django.db import models
 
-# Create your models here.
+class AssetType(models.Model):
+    """Stores all asset types"""
+    type = models.CharField(max_length=500)
```

Comment:
we might need to go with a different name since `type` is a built-in function in python..

Reply:
lets also have `created_at` and `last_modified` ([for ref](https://github.com/AndelaOSP/art-backend/blob/5024bb35614422762890243446f9bd334055a53e/art/api/models.py#L7))

Reply:
+1 to having `created_at` and `last_modified`